data/dat38_60.s

```diff
@@ -52856,8 +52856,10 @@ byte_8619B90:: .byte 0x60, 0x3, 0x60, 0x13, 0x60, 0x13, 0x71, 0x13, 0x72
 	.byte 0x13, 0x60, 0x3, 0x60, 0x3, 0x60, 0x13, 0x60, 0x13
 	.byte 0x7F, 0x13, 0x80, 0x13, 0x60, 0x13, 0x60, 0x13, 0x60
 	.byte 0x3
-byte_8619BD0:: .byte 0x0, 0x0, 0x0, 0x0, 0x94, 0x8, 0x0, 0x0, 0x98, 0x8, 0x0, 0x0, 0x90, 0x12
-	.byte 0x0, 0x0, 0x10, 0x78, 0x14, 0x0, 0x0, 0x16, 0x2, 0x0, 0x0, 0x8A, 0x66, 0x5C
+
+ACDCTownMapTriggers::
```

Comment:
Map Triggers/Wall Data shouldn't be included as GNU Assembler source, as they are binary files. You can keep these here for now, but remember that the end product would have these included as a custom binary format. I would also recommend reading [this post by GreigaMaster](https://forums.therockmanexezone.com/mmbn-map-wall-region-format-t16550.html#p352248) which goes into extensive detail about the Map Trigger/Wall Data/Map Region format.